### DISCUSSION TITLE
How do I execute for both JS and TS files at the same time

### DISCUSSION THREAD

**Discussion by brettz9** [TypeStrong/ts-node#1378] [Q&A]
Adding `allowJs` as per the README causes my coverage reporter (c8) to show only JavaScript files, whereas when I remove `allowJs`, I only see TypeScript files in the report (despite having both ".ts" and ".js" added to my Mocha `extensions`).

**Comment by parzhitsky** [comment]
I think, that might be more of a question to Mocha
